Sources/_CryptoExtras/RSA/RSA.swift

```diff
@@ -538,8 +538,8 @@ extension _RSA.Encryption {
         public var keySizeInBits: Int { self.backing.keySizeInBits }
         fileprivate init(_ backing: BackingPublicKey) { self.backing = backing }
 
-        public func getKeyPrimitives() throws -> Primitives {
-            let (n, e) = try self.backing.getKeyPrimitives()
+        public func getKeyPrimitives() -> Primitives {
```

Comment:
Let's leave the `throws` on the public functions and just remove the incorrect internal `try`s.